bottlerocket/testsys/src/run_aws_k8s.rs

```diff
@@ -373,7 +373,11 @@ impl RunAwsK8s {
             node_ami: self.ami.clone(),
             // TODO - configurable
             instance_count: Some(2),
-            instance_type: self.instance_type.clone(),
+            instance_types: self
+                .instance_type
+                .clone()
+                .map(|instance_type| vec![instance_type])
+                .unwrap_or_default(),
```

Comment:
I think this will also work, `self.instance_type.to_owned().iter().collect()`

Not a blocker but you can switch it over if you like it better.